.github/workflows/publish.yml

```diff
@@ -3,17 +3,25 @@ name: Publish
 on:
   push:
     tags:
-      - 'v*'
+      - "v*"
```

Comment:
Let's sync to make sure your prettier is setup properly.  If you run code formatter in VS Code, it should change this back to single quotes and remove the blank lines.